spacy/pipeline/pipe.pyx

```diff
@@ -81,6 +81,14 @@ cdef class Pipe:
 
         DOCS: https://spacy.io/api/pipe#score
         """
+        if hasattr(self, "scorer") and self.scorer is not None:
+            scorer_kwargs = {}
+            # use default settings from cfg (e.g., threshold)
+            if hasattr(self, "cfg") and isinstance(self.cfg, dict):
+                scorer_kwargs.update(self.cfg)
```

Comment:
This make sense. 

One exception here is that `self.cfg["labels"]` isn't always the same as `self.labels`. Maybe for robustness, it would make sense to account for that field specifically?

Something like
```
if hasattr(self, "labels"):
    scorer_kwargs["labels"] = self.labels
```